### PR TITLE
Origin/tv request type

### PR DESCRIPTION
**Pull request by rigrassm**
Change Log
==========
* Requests now show whether all or new episodes of a TV series were requested

* Removed lokenx/themoviedbsearch and rigrassm/tvmaze meteor package dependencies by consolidating their code into the application.

* Requests now show how many seasons that show has. This plus seeing the request type should make it easier to decide on approving or denying

* Automatic Updater script will go through and add the season count to requests in the users database that were entered prior to this version. This will run when the server restarts itself so you shouldn't see any real delay.

### CODE REVIEW

File: client/templates/requests/requests.html [hunk 1]
```diff
@@ -63,6 +63,9 @@
 							{{#if searchType 'TV Shows'}}
                            <h4> <li><a href="{{link}}" target="_blank"><strong>{{title}} ({{year}})</strong></a></li></h4>
 							{{/if}}
+						{{#if searchType 'TV Shows'}}
```
Comment (lokenx) [inline client/templates/requests/requests.html]: There's already a TV show conditional above and below this, combine with the one above or all 3?

File: server/methods/admin/updateDB.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 Meteor.methods({
 	'updatePosters': function() {
-		var results = TV.find({poster_path: /^\//}).fetch();
+		var results = TV.find({seasons: -1}).fetch();
 		for(i=0;i<results.length;i++){
-			TV.update({_id: results[i]._id}, { $set: {poster_path: "https://image.tmdb.org/t/p/w154"+results[i].poster_path}});
+            var response = HTTP.call("GET", "http://api.tvmaze.com/shows/" + results[i].id + "/seasons", {})
+            var seasons = response.data; 
+            console.log(results[i].id + " | " + results[i].title + " | " + seasons.length);
```
Comment (lokenx) [inline server/methods/admin/updateDB.js]: Debug log output? If not use the logger itself.

File: server/methods/admin/version.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ Meteor.methods({
     return "master";
   },
   getVersion: function () {
-    return "1.10.2";
+    return "1.10.1";
```
Comment (lokenx) [inline server/methods/admin/version.js]: This would be considered a new feature but not breaking so 1.11.0

File: version.txt [hunk 4]
```diff
@@ -1 +1 @@
-1.10.2
+1.10.1
```
Comment (lokenx) [inline version.txt]: 1.11.0 as above

File: server/methods/admin/updateDB.js (hunk 2 above)
Comment (rigrassm) [inline server/methods/admin/updateDB.js]: Was for testing but may be worth logging what's changed.

File: client/templates/requests/requests.html (hunk 1 above)
Comment (rigrassm) [inline client/templates/requests/requests.html]: Good catch, put it all under one roof.

**Pull request merged**